client/js/domhelpers.js

```diff
@@ -31,6 +31,12 @@ export function domByTemplate(id, type) {
   return div;
 }
 
+export function escapeCSS(id) {
+  if(!id)
+    return '';
+  return id.split('').map(s=>'\\' + s.charCodeAt(0).toString(16)).join('');
```

Comment:
I see you're just escaping every character rather than checking to see if the character requires an escape.

Reply:
Yeah. Think that's a problem? Code is 100x simpler.